src/data.js

```diff
@@ -1607,6 +1607,22 @@ module.exports = [
       'Visual Studio Code',
     ],
   },
+  {
+    name: 'Hideki Jinnai',
+    description: 'Lifelong Learner',
+    url: 'https://github.com/dekisr/uses',
+    emoji: '🧟',
+    country: '🇧🇷',
+    computer: 'apple',
+    phone: 'iphone',
+    tags: [
+      'Front...and...end',
```

Comment:
```suggestion
      'Front End',
```

You would not be found on the site when filter for the tag `Front End`, so this is preferred.